server/src/test/java/org/elasticsearch/cluster/routing/allocation/AddIncrementallyTests.java

```diff
@@ -154,7 +154,8 @@ public void testMinimalRelocationsNoLimit() {
             ClusterRebalanceAllocationDecider.ClusterRebalanceType.ALWAYS.toString()
         )
             .put("cluster.routing.allocation.node_concurrent_recoveries", 100)
-            .put("cluster.routing.allocation.node_initial_primaries_recoveries", 100);
+            .put("cluster.routing.allocation.node_initial_primaries_recoveries", 100)
```

Comment:
test case conditions assume previous default value of 2 for `cluster.routing.allocation.cluster_concurrent_rebalance`

planning to add new test for new behaviour as per new default .